packages/esm-ohri-pmtct-app/src/views/mch-summary/mch-summary.component.tsx

```diff
@@ -5,14 +5,29 @@ import styles from '../common.scss';
 import { PatientChartProps } from '@ohri/openmrs-esm-ohri-commons-lib';
 import CurrentPregnancy from './tabs/current-pregnancy.component';
 import HivExposedInfant from './tabs/hiv-exposed-infant.component';
-import { usePatient } from '@openmrs/esm-framework';
+import { usePatient, useConfig } from '@openmrs/esm-framework';
 import dayjs from 'dayjs';
 
 const MaternalSummary: React.FC<PatientChartProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
+  const { identifiersTypes } = useConfig();
   const { patient, isLoading } = usePatient(patientUuid);
   const dob = patient?.birthDate;
   const age = useMemo(() => dayjs().diff(dayjs(patient?.birthDate), 'year'), [patient?.birthDate]);
+  const [pTrackerId, setPtrackerId] = useState('');
+
+  useEffect(() => {
+    if (patient) {
+      const reversedIdentifiers = patient.identifier.slice().reverse();
```

Comment:
What is the functionality of `slice()` in this instance? Also, is there a reason why we are reversing this array?

Reply:
@lucyjemutai hanging review here...